cmd/openshift-install/gather.go

```diff
@@ -249,6 +249,9 @@ func extractHostAddresses(config *types.InstallConfig, tfstate *terraform.State)
 			return bootstrap, port, masters, err
 		}
 		masters, err = gatherkubevirt.ControlPlaneIPs(tfstate)
+		if err != nil {
+			return bootstrap, port, masters, err
```

Comment:
This should not be returning an error. The control planes may not be available, but we still want to gather from the bootstrap. We can log this error like it is done for other platforms, but we should not return the error.

Reply:
I've changed it.